### ISSUE TITLE
Cannot reorder icons since update to Gnome 45

### ISSUE THREAD

**Issue by berrfred** [micheleg/dash-to-dock#2122]
Recently upgraded a few laptops to FC39 / Gnome 45 and reordering icons seems to be impossible on all of them.

**Comment by vanvugt** [comment]
What version of dash-to-dock are you using?

**Comment by berrfred** [comment]
> What version of dash-to-dock are you using?

89

![image](https://github.com/micheleg/dash-to-dock/assets/50397929/add7dfa7-f5c1-458d-bcd9-629a3f4f5ed4)

**Comment by eboye** [comment]
For me it's actually this one

https://gitlab.gnome.org/harshadgavali/favourites-in-appgrid/-/issues/5

**Comment by berrfred** [comment]
You are right, if I disable **Favourites in AppGrid** and log out / log in, then it becomes possible to reorder icons in **Dash to Dock**.

**Comment by bclouthier** [comment]
Having the same issue. I was able to get around it by disabling Favourites in AppGrid and logging out and then moving the icons and logging back in and re-enabling the extension.

**Comment by bclouthier** [comment]
> Having the same issue. I was able to get around it by disabling Favourites in AppGrid and logging out and then moving the icons and logging back in and re-enabling the extension.

_That time you search for the same issue six months later and find your previous comment solving the same issue._

**Comment by brunos3d** [comment]
After researching the issue, I came across multiple discussions regarding the inability to display pinned apps in the AppGrid without affecting the layout of the Dash-to-Dock extension. This led me to fork the original [favourites-in-appgrid](https://gitlab.gnome.org/harshadgavali/favourites-in-appgrid/) extension (in which causes the re-arrangement issue) and implement a solution that would restore the visibility of pinned apps in both the GNOME Dock and Application Grid, without disrupting any existing Dock arrangements.

https://github.com/brunos3d/pinned-apps-in-appgrid

This repository aims to provide a simple yet effective fix to this issue, preserving the GNOME Dock layout while ensuring pinned apps remain visible in the grid.

**Comment by brunos3d** [comment]
Our extension is finally available on gnome extensions
https://extensions.gnome.org/extension/7660/keep-pinned-apps-in-appgrid/
